classes/User.php

```diff
@@ -481,10 +485,31 @@ public function loadData($UPDATE = User::UPDATE_FULL, $remote_view = false): str
             array_unshift($this->stats, 'bloodline_skill');
         }
 
-        $this->location = $user_data['location']; // x.y
-        $location = explode(".", $this->location);
-        $this->x = $location[0];
-        $this->y = $location[1];
+        if ($user_data['location'])
```

Comment:
Where should the `{ }` be for this if statement?

Reply:
Ah, I left this here. I was going to do a check to verify that the location should be in x:x:x format but the migration should be taking care of this.